### PR TITLE
#8 and updated versions

### PR DESCRIPTION
**Pull request by jeffmay**
@vodopis @chrissalij-r 

Just some regular maintenance. I plan to make the cut over to 2.11 only in a new major version release that will upgrade to Play 2.4.x line.

### CODE REVIEW

**Comment by JanPiper** [comment]
LGTM

File: build.sbt [hunk 1]
```diff
@@ -64,10 +64,10 @@ lazy val playJsonTests = project in file("playJsonTests") settings(common: _*) s
   name := "play-json-tests",
 
   libraryDependencies ++= Seq(
-    "com.typesafe.play" %% "play-json" % "2.3.7",
-    "org.scalacheck" %% "scalacheck" % "1.12.2",
-    "org.scalatest" %% "scalatest" % "2.2.4",
-    "me.jeffmay" %% "scalacheck-ops" % "1.0.0"
+    "com.typesafe.play" %% "play-json" % "2.3.9",
+    "org.scalacheck" %% "scalacheck" % "1.12.5",
+    "org.scalatest" %% "scalatest" % "2.2.6",
```
Comment (egprentice) [inline build.sbt:71]: not quite ready for scalatest 3.0 ?

File: build.sbt [hunk 2]
```diff
@@ -52,7 +52,7 @@ lazy val playJsonOps = project in file("playJsonOps") settings(common: _*) setti
   name := "play-json-ops",
 
   libraryDependencies ++= Seq(
-    "com.typesafe.play" %% "play-json" % "2.3.7"
+    "com.typesafe.play" %% "play-json" % "2.3.9"
```
Comment (jeffmay) [inline build.sbt]: This can be 2.3.10

**Comment by egprentice** [comment]
LGTM

File: build.sbt (hunk 1 above)
Comment (jeffmay) [inline build.sbt:71]: It's still a milestone and not a release. Also, these updates are going to be the latest on the 2.10 branch. After these changes I plan to remove support for 2.10 and upgrade everything to the latest major versions.

**Comment by egprentice** [comment]
ScalaCheck 1.13.0 was just released https://github.com/rickynils/scalacheck/blob/1.13.0/RELEASE

**Pull request merged**